### PR TITLE
Feat: Enable Copying Data from AG Grid to Excel - 620

### LINKED ISSUE #620: LCFS - Investigate Copying from AG Grid Cells to Excel
**Description:**

Investigate enabling direct copying of data from AG Grid cells into an Excel file within our frontend application. The goal is to provide users with an easy way to export data from AG Grid to Excel.

**Objectives:**

- Understand AG Grid's capabilities for data export.
- Explore libraries/tools for exporting data to Excel.
- Create a plan for implementing this feature.
- Develop a basic proof of concept if feasible.

**Acceptance Criteria:**

- Research AG Grid's export capabilities.
- Create implementation plan.
- Develop a simple proof of concept demonstrating data export to Excel.

**Tasks:**

1. **Research AG Grid Capabilities:**
   - Review AG Grid documentation for export options.

2. **Explore Libraries/Tools:**
   - Compare features and compatibility with React.js.

3. **Identify Implementation Steps:**
   - Outline steps to integrate the chosen library with AG Grid in React.js.

4. **Develop Proof of Concept:**
   - Implement functionality to copy data and export to Excel.

5. **Documentation and Next Steps:**
   - Document findings and provide recommendations for further development.

### PR DESCRIPTION
**Pull request by hamed-valiollahi**
This PR enables copying data directly from AG Grid cells to an Excel file.

Closes #620

### CODE REVIEW

**Review by AlexZorkin — COMMENTED** [review]
This is an interesting feature, but I wonder if its better just to use the backend download data feature. Let's ask our product owner if he thinks this has additional value.

**Comment by hamed-valiollahi** [comment]
> This is an interesting feature, but I wonder if its better just to use the backend download data feature. Let's ask our product owner if he thinks this has additional value.

Thank you for reviewing my PR. I agree with you. Using backend download to provide a full list of records makes more sense. Front-end download is useful for short lists that fit on one page with different filters applied.

File: frontend/src/views/Transactions/Transactions.jsx [hunk 1]
```diff
@@ -225,6 +225,8 @@ export const Transactions = () => {
           handleGridKey={handleGridKey}
           handleRowClicked={handleRowClicked}
           enableCopyButton={false}
+          enableExportButton={true}
```
Comment (AlexZorkin) [inline frontend/src/views/Transactions/Transactions.jsx:228]: Let's just remove this feature from transactions but can think about other tables we could use it.

**Review by AlexZorkin — CHANGES_REQUESTED** [review]

**Review by AlexZorkin — APPROVED** [review]

File: frontend/src/views/Transactions/Transactions.jsx (hunk 1 above)
Comment (hamed-valiollahi) [reply frontend/src/views/Transactions/Transactions.jsx]: Sure, thank you for reviewing my PR.

**Pull request merged**